miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -845,6 +866,10 @@ public ModelAndView editPropagateSamples(@PathVariable String sampleIds, ModelMa
           hasPlain = true;
         }
         LibraryDto library = new LibraryDto();
+        Library tempLibrary = new LibraryImpl();
+        tempLibrary.setSample(sample);
+        final String libraryAlias = namingScheme.generateLibraryAlias(tempLibrary);
```

Comment:
it is I'm afraid.  Without an alias the form is invalid so we need to know it upfront.